src/fourcipp/fourc_input.py

```diff
@@ -65,6 +66,42 @@ def is_section_known(section_name: str) -> bool:
 CONVERTER = Converter()
 
 
+def modify_input_with_defaults(input_path: pathlib.Path, overwrite_flag: bool) -> None:
+    """Apply user defaults to an input file located at input_path.
+
+    Args:
+         input_path: Input filename to apply user defaults to.
+         overwrite_flag: Whether to overwrite the existing input file.
+                         By default, a new file with suffix '_mod.4C.yaml' is created.
+    """
+    OUTPUT_SUFFIX = "_mod.4C.yaml"
```

Comment:
```suggestion
    output_suffix = "_mod"
```
The ending of the file should be extracted later. Also, `output_suffix` should be lowercase.

Reply:
I don't know what it's good for to separate the suffix, but if you say  so. I suggest to add `.4C.yaml` even if the input file did not have this suffix.